queries/typoscript/highlights.scm

```diff
@@ -0,0 +1,48 @@
+(identifier) @field
```

Comment:
are all identifiers a field? seems a bit weird but that could be the case after all

Reply:
Yes imho every identifier is a field. A TypoScript configuration is a hierarchical collection of configuration blocks (allthough "configuration" is maybe not the correct word). Each block has an identifier and contains fields. Each of this fields can also be a block of configuration items. The identifier can be the identifier of the block itself or a path in the configuration tree. So for example

```
foo {
    bar {
        test = 1
    }
}
```
is the same as `foo.bar.test = 1`
So in my opinion `foo` and `bar` and `test` are fields inside the configuration. Maybe you find `@property` a better match?

Reply:
thanks! field is good as is